h/views/api_groups.py

```diff
@@ -16,9 +16,23 @@ def groups(request):
     authority = request.params.get('authority')
     document_uri = request.params.get('document_uri')
     svc = request.find_service(name='list_groups')
-    all_groups = svc.all_groups(user=request.user,
-                                authority=authority,
-                                document_uri=document_uri)
+
+    if request.feature('filter_groups_by_scope'):
```

Comment:
Note that the feature flag merely "protects" the view from using the scope filtering — the actual new filtering method, `request_groups`, in `ListGroupsService` can be used without the feature flag being enabled, but the view is currently the only place it's used. If this makes people feel wiggly, we could adjust this.